apps/server/src/shared/repo/task/task.repo.integration.spec.ts

```diff
@@ -530,6 +530,26 @@ describe('TaskRepo', () => {
 				expect(result[2].id).toEqual(task3.id);
 				expect(result[3].id).toEqual(task4.id);
 			});
+
+			it('should order by id asc when no dueDate is set', async () => {
+				const teacher = userFactory.build();
+				const tasks: Task[] = [];
+				for (let i = 0; i < 30; i += 1) {
```

Comment:
Should buildList not do the same in the end, with same order? const tasks = taskFactory.buildList(30, { creator: teacher, dueDate: undefined }); ?

Reply:
True. I updated that.